src/transforms/pull-request.ts

```diff
@@ -48,11 +49,11 @@ function mapReviews(reviews: Octokit.PullsListReviewsResponse = []) {
 
 // TODO: define arguments and return
 export default async (github: GitHubAPI, pullRequest: Octokit.PullsGetResponse, reviews?: Octokit.PullsListReviewsResponse, log?: LoggerWithTarget) => {
-	const { title: prTitle, head } = pullRequest;
+	const { title: prTitle, head, body } = pullRequest;
+
 	// This is the same thing we do in sync, concatenating these values
-	const issueKeys = issueKeyParser().parse(
-		`${prTitle}\n${pullRequest.head.ref}`
-	);
+	const textToSearch = await booleanFlag(BooleanFlags.ASSOCIATE_PR_TO_ISSUES_IN_BODY, true) ? `${prTitle}\n${head.ref}\n${body}}` : `${prTitle}\n${head.ref}`
```

Comment:
I changed the old one to use `head.ref` instead, as that was already extracted before

Reply:
Do we need to change old code if we are going to roll out the feature flag anyway? Would be safer to keep it unchanged

Reply:
I agree. Better to leave old code unchanged and create a follow-up ticket to implement this way once FF is completely rolled out.